src/ui/components/AccountSelector.js

```diff
@@ -27,24 +26,46 @@ export default function AccountDropdown({
   placeholder,
   setCurrentIdentity,
   ledger,
+  balance,
 }: DropdownProps) {
   const items = ledger.accounts().filter((a) => a.active);
+  const [selectedAccount, setSelectedAccount] = useState<Account | null>(null);
```

Comment:
Looks like we can either set the balance explicitly as a prop, or we can get it implicitly from the account. Why both? I'd prefer to have a single source of truth for the balance (e.g. the account)

Reply:
Yes you are right. I came to this solution in a roundabout way trying to fix the balance updating issue topocount saw, but after adding this selectedAccount state line, the balance prop is redundant.